templates/CRM/Admin/Form/Setting/Search.tpl

```diff
@@ -68,6 +68,14 @@
             <td>{$form.smartGroupCacheTimeout.html}<br />
                 <span class="description">{ts}The number of minutes to cache smart group contacts. We strongly recommend that this value be greater than zero, since a value of zero means no caching at all. If your contact data changes frequently, you should set this value to at least 5 minutes.{/ts}</span></td>
         </tr>
+        <tr class="crm-search-setting-form-block-quicksearch_options">
+            <td class="label">{$form.quicksearch_options.label}</td>
+            <td>
+                {$form.quicksearch_options.html}
+                <p class="description">{ts}Which fields can be searched on in the menubar quicksearch box?{/ts}</p>
```

Comment:
Use the description from the metadata - it IS assigned to the tpl I'm pretty sure

Reply:
Ok, but I notice that the description field is not wrapped in `ts()` in Search.setting.php for any field. Is that okay?